.travis.yml

```diff
@@ -0,0 +1,56 @@
+# Used old infrastructure, needed for integration tests:
+# http://docs.travis-ci.com/user/workers/standard-infrastructure/
+sudo: required
+language: python
+python:
+  - 2.7
+
+branches:
+  only:
+    - master
+
+env:
+  global:
+    - CACHE_NAME=JOB1
+  matrix:
+    - TASK=ci-unit NODE_INDEX=0 NODE_TOTAL=2
+    - TASK=ci-unit NODE_INDEX=1 NODE_TOTAL=2
+    - TASK=ci-integration
+    - TASK=ci-checks
+    - TASK=ci-packs-tests
+
+addons:
+  apt:
+    sources:
+      - mongodb-upstart
+      - sourceline: 'deb [arch=amd64] http://repo.mongodb.org/apt/ubuntu precise/mongodb-org/3.4 multiverse'
+        key_url: 'https://www.mongodb.org/static/pgp/server-3.4.asc'
+    packages:
+      - mongodb-org-server
+      - mongodb-org-shell
+
+services:
+  - mongodb
+  - rabbitmq
+
+cache:
+  pip: true
+  directories:
+    - virtualenv/
```

Comment:
Yeah, we just need to be careful, we had a lot of issues with this in the past - e.g. when Travis upgraded Python version or some system library on which some Python library with C extension depends on.

So if similar issues appear again, we will need to wipe this directory and re-create it on Travis.